README.md

```diff
@@ -32,8 +32,8 @@ Welcome to the Better Bar Crawl App! This application allows users to discover a
 ### Clone the Repository
 
 ```bash
-git clone https://github.com/yourusername/BetterBarCrawlApp.git
-cd BetterBarCrawlApp
+git clone https://github.com/pwjensen/BetterBarCrawlApp.git
+cd bar-crawl-app
```

Comment:
Incorrect I think

Reply:
Line 36

Reply:
![image](https://github.com/user-attachments/assets/5eba3f2f-f829-47b7-815d-4afb508b7060)
![image](https://github.com/user-attachments/assets/cc44cdf8-d52f-44d8-a561-23f4d0310f1d)

Reply:
![image](https://github.com/user-attachments/assets/d72e9164-a0e9-4fe2-8cf3-84e54942f2c9)

Reply:
unless other git implementations do some wonky stuff

Reply:
so line 36 is incorrect, if it's added

Reply:
me when im stupid